src/symbols/Node/Node.tsx

```diff
@@ -174,6 +176,16 @@ export const Node: FC<Partial<NodeProps>> = ({
     });
   }, [controls, x, y]);
 
+  const nodeChildProps: NodeChildProps = {
```

Comment:
@amcdnl Refactored this to avoid passing props that might not respect the NodeChildProps interface (better TS types)